builder/proxmox/common/config.go

```diff
@@ -346,6 +352,24 @@ func (c *Config) Prepare(upper interface{}, raws ...interface{}) ([]string, []st
 			errs = packersdk.MultiErrorAppend(errs, fmt.Errorf("one of iso_file, iso_url, or a combination of cd_files and cd_content must be specified for AdditionalISO file %s", c.AdditionalISOFiles[idx].Device))
 		}
 	}
+	if c.EFIDisk != "" {
+		if c.EFIConfig != (efiConfig{}) {
+			errs = packersdk.MultiErrorAppend(errs, fmt.Errorf("both efi_config and efidisk set, remove deprecated efidisk"))
```

Comment:
Nit regarding phrasing here: 
```suggestion
			errs = packersdk.MultiErrorAppend(errs, fmt.Errorf("both efi_config and efidisk cannot be set at the same time, consider defining only efi_config as efidisk is deprecated"))
```

Reply:
Done, thanks!